src/components/form/question/ChoicesInput.tsx

```diff
@@ -23,12 +23,22 @@ export default function ChoicesInput(props: Props) {
     firstInputId,
     ...otherProps
   } = props
+
+  // For now, it only concerns `DPE` question whose possibilities are very short, so 4 colomns is ok. However, we should have done a special question.
+  const shouldUseGridTreshold = 6
```

Comment:
Encore une règle à la mort moi le noeud mais quand on définit une constante globale, c'est une bonne pratique de la passer en majuscules, ça permet de savoir en un coup d'oeil que c'est une constante qui ne bougera pas

```suggestion
  const SHOULD_USE_GRID_COLUMN = 6
```